### DISCUSSION TITLE
靈魂之窗守護者，美科學家研發 24 小時追蹤眼壓的智慧隱形眼鏡

### DISCUSSION THREAD

**Discussion by neil0258** [1101-GS3073-C/Python-Community-GS3073-C#675] [General]
青光眼患者常飽受眼壓所苦，一旦眼壓升高沒有好好治療，便可能造成視力喪失。近期美國科學家研發出新型智慧隱形眼鏡，患者戴上後 24 小時內追蹤眼壓變化，幫助患者與眼科醫生掌握靈魂之窗狀態，給予最適當的醫療處置。
眼壓升高時，可能會頭痛、噁心、眼睛脹痛和視力模糊，需到醫院看診檢測，才會知道確切的眼壓變化。但如果有智慧隱形眼鏡，戴上後隨時追蹤眼壓，會想戴看看嗎？
我的看法：現在每天期末考與感覺讀書讀到眼睛快爆炸了，買了這種測試眼壓的隱形眼鏡給教授看看我的眼壓，看看能不能規定眼壓超過多少以上的，可以加個醫療分之類的。累累病
https://technews.tw/2022/10/23/smart-contacts-monitor-eyes-for-sign-of-glaucoma-247/

**Comment by liangyinglly** [comment]
身邊有人也是飽受創傷性青光眼所苦，相信這個發明一定能幫助到他以經濟實惠價格及更舒適的方式監控眼睛健康！

**Comment by 0212Jerry** [comment]
這副隱形眼鏡在安全、無副作用的情況下，個人認為這會是一項很好的發明，此外，這項產品也有很大的發展性，除了追蹤青光眼之外，也可能擴充其他功能，如追蹤白內障等眼部疾病或自動偵測外部光線強度而自動調整成眼睛能適應的亮度，類似自動太陽眼鏡的功能。以上都可以大幅增加實用性和普遍性。

**Comment by yjpxi** [comment]
如果這種技術能夠普及，甚至可能成為學校或工作場所的健康管理工具，幫助學生和員工更好地管理視力健康
